docs/getting_started.rst

```diff
@@ -0,0 +1,358 @@
+.. don't forget to set permissions hyperlink
+  + commands links + guide links
+
+.. _getting-started:
+
+===============
+Getting started
+===============
+
+If you recently installed Red, you should read this.
+This is a quick start guide for a general usage.
+
+.. note::
+
+    If you haven't installed Red, please do it by following
+    the :ref:`installation guides <main>`.
+
+Assuming you correctly installed Red, you should have a
+window like this:
+
+.. image:: .resources/red-console.png
+
+.. _gettings-started-invite:
+
+-------------------------
+Invite Red to your server
+-------------------------
+
+When started, the console will show you ``Invite URL`` (here at
+the bottom of the screenshot).
+Paste the link into your browser and select the server you want
+to invite the bot in, like any other bot.
+
+.. note:: You need the ``Manage server`` permission to add bots.
+
+Complete the captcha, it should tell you ``Authorized!`` and you
+should see your bot in the members list.
+
+.. attention::
+    If Discord shows ``Bot requires code grant``, please untick this
+    option in your token settings
+
+    .. image:: .resources/code-grant.png
+
+.. _getting-started-interact:
+
+-----------------
+Interact with Red
+-----------------
+
+As a chatbot, you interact with Red via the Discord text channels
+(not from the command prompt). For that, you will use the prefix you
+set before. For example, if your prefix is ``!``, you will execute your
+command like this: ``!ping``.
+
+.. note:: Since the prefix can be anything, it'll be referenced as ``[p]``
+    in documentations.
+
+.. _getting-started-commands:
+
+~~~~~~~~~~~~
+The commands
+~~~~~~~~~~~~
+
+The command you're going to use the most is help. That command will
+show you **all of the available commands** of the bot with a small description.
+
+.. code-block:: none
+
+    [p]help
+
+.. tip:: The message is generated dynamically and users will only see the
+    commands they can use. You can change what commands users can use with the
+    permissions cog.
+
+You can also pick a command to get its detailed description and the
+parameters.
+
+.. code-block:: none
+
+    [p]help command
+
+The parameters are shown as enclosed in ``< >`` if they're needed, or
+``[ ]`` if optional.
+As an example, the ban command will show this in the help message:
+``Syntax: !ban <user> [days] [reason]``
+
+This means that it is necessary to provide ``user``. However, the
+``days`` value (number of messages to delete) is optional, like
+the ``reason`` value, used for the modlog.
+
+You can use help to show the **categories** too (named cogs).
```

Comment:
"named cogs" is a tad confusing, maybe change it to "also called cogs". The parentheses could also be removed.